src/components/item/RelatedUnits.vue

```diff
@@ -0,0 +1,437 @@
+      <div 
+        v-for="item in currentItemObj?.relatedItems"
+        :key="item.id"
+        class="related-unit-container"
+      >
+        <ButtonSign
+          class="delete-related-unit-button"
+          style-type="cross"
+          title="delete"
+          @click="deleteRelatedItem(item.id)"
+        />
+        <router-link
+          :to="{ name: 'item', params: { id: item.id || item } }"
+          class="related-unit"
+          target="_blank"
+        >
+          {{ item.title }}
+        </router-link>
+      </div>
+    </div>
+    <div v-if="currentItemObj?.relatedLists?.length">
+      <h1>related lists:</h1>
+      <div 
+        v-for="list in currentItemObj?.relatedLists"
+        :key="list.id"
+        class="related-unit-container"
+      >
+        <ButtonSign
+          class="delete-related-unit-button"
+          style-type="cross"
+          title="delete"
+          @click="deleteRelatedList(list.id)"
+        />
+        <router-link
+          :to="{ name: 'list', params: { id: list.id || list } }"
+          class="related-unit"
+          target="_blank"
+        >
+          {{ list.title }}
+        </router-link>
+      </div>
+    </div>
+    <div class="single-button-container">
+      <ButtonText
+        text="add related list or item"
+        style-type="underline"
+        @click="toggleShowingStatus('addRelatedForm')"
+      />
+    </div>
+    <div 
+      v-if="showingStatuses.addRelatedForm"
+      class="add-related-form"
+      :class="{ bordered: !isItemFormInSidebar}"
+    >
+      <div class="related-unit-type">
+        <RadioCustom 
+          v-for="relatedUnitType in ['list', 'item']"
+          :key="relatedUnitType"
+          :label="relatedUnitType"
+          :value="relatedUnitType"
+          :model-value="newRelatedUnitType"
+          style-type="initial"
+          small
+          @update:model-value="value => changeNewRelatedUnitType(value)"
+        />
+      </div>
+      <div class="unit-options">
+        <SelectCustom 
+          :default-option="$options.LISTS_DEFAULT_OPTION"
+          :default-option-selected="!chosenListId"
+          :disabled="areTextFieldsEmpty"
+          :model-value="chosenListId"
+          @update:model-value="value => setNewRelatedUnitId('list', value)"
+        >
+          <option
+            v-for="list in choosableLists"
+            :key="list.id"
+            :value="list.id"
+          >
+            {{ list.title }}
+          </option>
+          <optgroup
+            v-if="newRelatedUnitType === 'list' && edittingItemObj.relatedLists?.length"
+            label="already related lists:"
+            disabled
+          >
+            <option
+              v-for="list in currentItemObj?.relatedLists"
+              :key="list.id"
+            >
+              {{ list.title }}
+            </option>
+          </optgroup>
+        </SelectCustom>
+        <SelectCustom
+          v-show="newRelatedUnitType === 'item'"
```

Comment:
why v-show but not v-if?

Reply:
so that it was rendered and ready to appear